api/server/sdk/server.go

```diff
@@ -147,6 +151,7 @@ func (s *Server) Start() error {
 		api.RegisterOpenStorageIdentityServer(grpcServer, s.identityServer)
 		api.RegisterOpenStorageMountAttachServer(grpcServer, s.volumeServer)
 		api.RegisterOpenStorageAlertsServer(grpcServer, s.alertsServer)
+		api.RegisterOpenStorageClusterPairServer(grpcServer, s.clusterPairServer)
 	})
```

Comment:
You need to add the new service to the REST gateway. https://github.com/libopenstorage/openstorage/blob/master/api/server/sdk/server.go#L293

Reply:
Added the following
RegisterOpenStorageClusterPairHandlerFromEndpoint(